examples/nas/cream/requirements.txt

```diff
@@ -0,0 +1,12 @@
+yacs
+numpy==1.17
+opencv-python==4.0.1.24
+torchvision==0.2.1
+thop
+git+https://github.com/sovrasov/flops-counter.pytorch.git
+pillow==6.1.0
+torch==1.2
+timm==0.1.20
+tensorboardx==1.2
+git+https://github.com/Tramac/torchscope.git
```

Comment:
Can it be installed via pip torchscope?

Reply:
No, 'pip install torchscope' would throw an error.